netpyne_ui/netpyne_geppetto.py

```diff
@@ -340,17 +340,15 @@ def _prepare_simulation_files(self, experiment: model.Experiment = None, use_pre
 
     def _prepare_batch_files(self, experiment: model.Experiment) -> str:
         # param path must be split: [ {'label': ['synMechParams', 'AMPA', 'gmax']} ]
-        # TODO: how do we handle array index? ['array', '[0]'] or ['array', '0']?
         exp =  copy.deepcopy(experiment)
         exp.params = [p for p in exp.params if p.mapsTo != '']
 
         for param in exp.params:
             if param.type == "range":
                 param.values = list(np.arange(param.min, param.max, param.step))
             elif param.type == "list":
-                # TODO: need to enforce correct type for each parameter
-                #   e.g. numCells with 10.0 fails because it requires int not float
-                param.values = [int(e) for e in param.values]
+                # nothing to do here
+                pass
```

Comment:
why conversion is not needed anymore?

Reply:
because the frontend is doing now the validation and conversion, I just used here `int()` as workaround to be able to test the experiment flow